test/slot_test.exs

```diff
@@ -620,12 +581,7 @@ defmodule Surface.SlotTest do
       end
 
     message = """
-    code:1: there's no `default` slot defined in `Surface.SlotTest.OuterWithoutDefaultSlot`.
```

Comment:
Unfortunately we loose this proper error message as the compiler will detect that the `default` slot is not defined before looking at the `:let` directive.